src/styles/style.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;700;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: lato, sans-serif;
+  color: white;
+  background-color: #3445e0;
+  font-weight: 700;
+}
+
+.the-navbar {
+  display: flex;
+  padding: 10px;
+  justify-content: space-between;
+}
+
+.set-mic {
+  display: flex;
+  width: 20%;
+  justify-content: space-around;
+}
```

Comment:
- [ ] Kindly improve the design of your Hope page as well to match the original one (Typography: font face, size, background, color, weight...)👇🏻 

- Kindly check this [link](https://pierrebarba.github.io/capstone-metrics/) for some inspiration 👍🏻  you should improve such the background colors where one card has a background with `light` color and the other one in `dark`. Color, text color, .... and make sure to respect the design layout 🎨 🖌️ 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/221357382-6bd83b63-296e-418c-a438-e5b022ab07e0.png) | ![Screenshot 2023-02-09 at 10 15 21 (2)](https://user-images.githubusercontent.com/64774936/217769278-651d6695-ed6d-4197-9584-53be3452a033.png)

Reply:
we were asked to choose an api to work on and i picked this and designed it following the recommended design so why are you requesting a change, all functionality works

Reply:
- [ ] Hi @Outkarm! The external API is to have data you can display on your pages, however, according to the design requirement:

![image](https://user-images.githubusercontent.com/77069011/221369107-72f482f6-c56b-4a37-a40a-f351658c7383.png)

According to the design, these are the possible colors and fonts:

![image](https://user-images.githubusercontent.com/77069011/221369223-bef67be4-996f-4179-bf9e-90bd1da74887.png)

You have the font right, however, the colors don't match. Please consider changing it to meet this requirement! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/221369335-7a0335d3-7303-4929-8bcd-09f2bfbdc75d.png) | ![image](https://user-images.githubusercontent.com/77069011/221369461-286757cc-342b-462a-8fb7-54e94d9b2706.png) |
| ![image](https://user-images.githubusercontent.com/77069011/221369479-434dc0b4-36a9-48d7-b867-a9e1877b1968.png) | ![image](https://user-images.githubusercontent.com/77069011/221369448-d5ac6cce-8036-4e8f-a178-8e944352e8ee.png) |



Also, If you look closely, in your home page, the elements, in your case teams have a "chess-like" color design, some of them are light, others are darker. Please consider applying this as suggested by the previous reviewer! 🤓 🔧